inst/css/picker_input.css

```diff
@@ -1,5 +0,0 @@
-
-/* to set the alignment of pickerInput dropdowns */
-.dropdown-menu.open {
```

Comment:
hmm - so the pickerInput is inside optionalselectInput which lives in teal.widgets so not sure it should have been moved to teal?

Reply:
you are right, I will attach it directly in the optionalselectInput